docs/best-known-configuration.md

```diff
@@ -0,0 +1,102 @@
+# Best Known Configuration
+
+## Introduction
+
+Component `<tag>`s are used both by OEMs and by customers to identify a *known-working* (or
+commercially supported) set of firmware on the machine.
+
+This allows two things:
+
+* Factory recovery where a system in the field has been upgraded
+* Ensuring a consistent set of vendor-tested firmware for a specific workload
+
+The tags are either assigned in the firmware cabinet archive (the `.metainfo.xml` file) or added
+post-upload on the LVFS and are then included in the public AppStream metadata.
+A single firmware can be marked with multiple tags, and tags can be duplicated for different
+firmwares.
+This would allow an OEM to say *this set of firmware has been tested as a set for workload A,
+and this other set of firmware has been tested for workload B* which is fairly typical for
+enterprise deployments.
+
+## LVFS
+
+The LVFS added support for “vendor-defined” component `<tag>`s in 2021, which are also supported in
+fwupd since version 1.7.3.
+These tags are typically used by server vendors to identify a manifest of firmware for a specific
```

Comment:
still don't think this should have server in it

Reply:
Oopa, removed.